### PR TITLE
removed major and critical issues

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: Console/Command/UpgradeToSectionHasChildrenCommand.php [hunk 1]
```diff
@@ -162,8 +162,8 @@ private function updateSections()
         $sections = $this->repo->findAll();
         foreach ($sections as $section) {
             PageListener::setSectionHasChildren($this->em, $section);
-            $this->em->flush();
         }
+        $this->em->flush();
```
Comment (eric-chau) [inline Console/Command/UpgradeToSectionHasChildrenCommand.php]: ping @crouillon  on it, do you see any issue if we did not flush on every loop but only at the end?

Comment (ndufreche) [inline Console/Command/UpgradeToSectionHasChildrenCommand.php]: This is depending the project

**Comment by ndufreche** [comment]
:+1:

File: Console/Command/UpgradeToSectionHasChildrenCommand.php (hunk 1 above)
Comment (mickaelandrieu) [inline Console/Command/UpgradeToSectionHasChildrenCommand.php]: from my tests, nothing wrong happens with this update.

**Pull request merged**